index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Please add code linter from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) config.
Pull Request should have a short summary describing introduced changes in more detail.